engine/lib/navigation.php

```diff
@@ -318,6 +321,26 @@ function elgg_get_breadcrumbs(array $breadcrumbs = null) {
 		_elgg_services()->logger->error('"prepare, breadcrumbs" hook must return an array of breadcrumbs');
 		return [];
 	}
+	
+	foreach ($breadcrumbs as $key => $breadcrumb) {
+		if (isset($breadcrumb['link'])) {
+			elgg_deprecated_notice("Breadcrumb [{$breadcrumb['title']}] requires 'href' instead of 'link' set in the configuration", '3.0');
```

Comment:
are you sure `$breadcrumb['title']` is present, maybe do the title check first and then use `$breadcrumb['text']`?